centrallix-lib/src/qprintf.c

```diff
@@ -1035,6 +1038,12 @@ qpfPrintf_va_internal(pQPSession s, char** str, size_t* size, qpf_grow_fn_t grow
 			    strval = tmpbuf;
 			    break;
 
+            case QPF_SPEC_T_LL:
+                llval = va_arg(ap, long long);
+                cplen = snprintf(tmpbuf, sizeof(tmpbuf), "%lld", llval);
```

Comment:
I can't think of any situation where a ≤ 0 return value would actually happen for %lld, but it'd be good form to check the return value from snprintf(), to ensure it is positive.

Reply:
n/m it looks like the code already checks down below the switch.